packages/core/src/Provider/AssetProvider/types.ts

```diff
@@ -5,15 +5,30 @@ interface AssetExtraData {
   tokenMetadata?: boolean;
   history?: boolean;
 }
-export interface GetAssetArgs {
-  assetId: Cardano.AssetId;
+
+export interface GetAssetsCommonArgs {
   extraData?: AssetExtraData;
 }
 
+export interface GetAssetArgs extends GetAssetsCommonArgs {
+  assetId: Cardano.AssetId;
+}
+
+export interface GetAssetsArgs extends GetAssetsCommonArgs {
```

Comment:
Let's get rid of this unused option in the new endpoint
```suggestion
export interface GetAssetsArgs extends Omit<GetAssetsCommonArgs, 'history'> {
```

Reply:
> Please mark `get-asset` as deprecated in the OpenApi spec

[df87106](https://github.com/input-output-hk/cardano-js-sdk/pull/691/commits/df87106d7e81a99cc450371b1a6eed6b5e1d0224)

Reply:
done here [5ff5186](https://github.com/input-output-hk/cardano-js-sdk/pull/691/commits/5ff518689f4c8aff24347b00cc567566376e8dbc)